tensorboard/webapp/metrics/views/right_pane/right_pane_test.ts

```diff
@@ -89,6 +89,7 @@ describe('metrics right_pane', () => {
       );
       store.overrideSelector(selectors.getIsFeatureFlagsLoaded, true);
       store.overrideSelector(selectors.getIsMetricsImageSupportEnabled, true);
+      store.overrideSelector(selectors.getEnabledCardWidthSetting, false);
```

Comment:
Could you move this down a line (line 93) to make it more organized? (I would prefer alphabetically sorting but that's already not the case)

Reply:
Done.